src/ansys/rocky/core/rocky_api_proxies.py

```diff
@@ -165,3 +165,16 @@ def serialize(cls, obj: "ApiExportToolkitProxy") -> dict:
             "__class__": cls.__name__,
             "_session_uid": obj._session_uid if obj._session_uid is not None else None,
         }
+
+
+class ApiProjectProxy(ApiElementProxy):
+    def __init__(self, pyro_api: Pyro5.api.Proxy, session_uid: str | None = None) -> None:
+        super().__init__(pyro_api, 'project', session_uid)
+
+    def CloseProject(self, check_save_state: bool = True) -> None:
```

Comment:
I guess we need to also change the `RockyClient.close` signature to keep both consistent.